recipes/mne-python_1.0/construct.yaml

```diff
@@ -56,8 +56,8 @@ specs:
   - jinja2 <3.1,>=2.10  # Remove once https://github.com/conda-forge/numpydoc-feedstock/pull/17 has been merged
   # TODO: ⛔️ ⛔️ ⛔️ REMOVE ASAP STOP ⛔️ ⛔️ ⛔️
 
-  - mne =1.0.1=*_0
```

Comment:
Looks like our `1.0.1_1` installer was actually using build 0 :(

@hoechenberger do you see a way to add a test that would have caught this? Can we `grep` the output json for example to make sure that the `mne` and `mne-installer-menus` that are used have the correct `version`?

Reply:
Oh never mind I think we're okay on the 1.0.1 actually. But it still would have been nice to catch this error in this PR!